### ISSUE TITLE
Question about replication on waymo dataset 

### ISSUE THREAD

**Issue by lllcccfff** [fudan-zvg/4d-gaussian-splatting#40]
作者您好，我随机挑了一两个waymo场景训练的效果并不是很理想。有两个问题想向您请教。

1. 我尝试了选择第一帧的lidar点云和随机采样所有帧的lidar点云两种点云初始化方式，想问一下你们选取初始点云的方式。
2. 论文中seg0172120，seg0047140，seg0047140，seg0177030这四个片段我在waymo preception dataset的云盘里没找到，我该去哪里找呢？

谢谢！

**Comment by AutonomierLong** [comment]
I'm wondering whether you encountered the situation that the number of gaussians exploded, so that CUDA is out of memory? How can this be fixed?

**Comment by lllcccfff** [comment]
@AutonomierLong Yes, i have encounter it before in urban scene. I suggest you set the camera extent to a constant value that is independent of urban data. I set it to 200, and gaussians would not explode.
